docs/scos/dev/guidelines/project-development-guidelines.md

```diff
@@ -50,11 +55,22 @@ Additionally, we recommend keeping modules up to date in general.
 
```

Comment:
Can we add a new point here:

## Do not use/extend Private API

Check definition of [Private API](https://docs.spryker.com/docs/scos/dev/architecture/module-api/declaration-of-module-apis-public-and-private.html).
Do not use or extend Private API.
Register the missing extension point in [Spryker Ideas](https://spryker.ideas.aha.io/), so we add it in future and you can use Public API.